scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -60,7 +60,9 @@ public static BuildTargetResult FinalizeBuild(BuildTargetContext c)
             {
                 string targetContainer = $"{Channel}/Binaries/Latest/";
                 string targetVersionFile = $"{targetContainer}{CliNuGetVersion}";
-                string leaseId = AzurePublisherTool.AcquireLeaseOnBlob(targetContainer);
+                string semaphoreBlob = $"{Channel}/Binaries/publishSemaphore";
```

Comment:
I don't believe in this code path; we should only be touching the binaries/Latest folder here. Adding a semaphore for the installer binaries would be out-of-scope for this PR